CodewarsReporter.cfc

```diff
@@ -36,16 +38,25 @@ component {
 	 * @bundleStats Bundle stats
 	 * @print The print Buffer
 	 */
-	function genSuiteReport( required suiteStats, required bundleStats, required print ){
+	function genSuiteReport( required suiteStats, required bundleStats, required print, debugMap={}, labelPrefix='' ){
+		labelPrefix &= '/' & arguments.suiteStats.name;
 		print.line( prependLF( "<DESCRIBE::>#arguments.suiteStats.name#" ) );
 
 		for ( local.thisSpec in arguments.suiteStats.specStats ) {
+			var thisSpecLabel = labelPrefix & '/' & local.thisSpec.name;
 			print.line( prependLF( "<IT::>#local.thisSpec.name#" ) );
 
+			if( debugMap.keyExists( thisSpecLabel ) ) {
+				print.line( debugMap[ thisSpecLabel ] )
+			}
+
 			if ( local.thisSpec.status == "passed" ) {
 				print.line( prependLF( "<PASSED::>Test Passed" ) );
 			} else if ( local.thisSpec.status == "failed" ) {
 				print.line( prependLF( "<FAILED::>#escapeLF( local.thisSpec.failMessage )#" ) );
+			} else if ( local.thisSpec.status == "skipped" ) {
+				print.line( prependLF( "<LOG::>Skipped" ) );
+				print.line( prependLF( "<ERROR::>Test skipped" ) );
```

Comment:
This line can be removed. When test result (passed, failed, error) is not found, it marks the test case as failed. So just `<LOG::>Skipped` is enough to produce:

![image](https://user-images.githubusercontent.com/639336/73513700-7501bd80-43a3-11ea-8cd1-5e929dc3f741.png)

Having `<ERROR::>` will be too much and redundant:

![image](https://user-images.githubusercontent.com/639336/73513793-ea6d8e00-43a3-11ea-8575-1a7385672bf8.png)

Actually, can you replace these two lines with just `\n<FAILED::>Test Skipped`? I think it looks the best and it's more clear:
![image](https://user-images.githubusercontent.com/639336/73513827-05d89900-43a4-11ea-8fa7-83209a895fa5.png)

Reply:
So when I first tested just having the log line in your test kumite, I didn't think it had properly failed.  I was a little confused by the output as there were no test failure or success messages at the top but there was a green bar next to the text "Test Results" so it seemed like it had passed.
![image](https://user-images.githubusercontent.com/584009/73515490-1c3e1f00-43bb-11ea-966f-5331eda70291.png)
So that's why I went back and added the error bit in.  

I'll push a commit now that just uses the failed placeholder.